packages/driver/src/cy/multi-domain/validator.ts

```diff
@@ -21,13 +21,20 @@ export class Validator {
       })
     }
 
-    if (data && !Array.isArray(data)) {
+    if (data && (!isObject(data) || Array.isArray(data))) {
```

Comment:
```suggestion
    if (args && (!isObject(args) || Array.isArray(args))) {
```

Needs to be changed here and in the other places in this file.

Reply:
I think this one we want to be the full `options` object to make sure the `args` key is specified as well as to validate any other options in the future. updated `data` to `options` in [b06f216](https://github.com/cypress-io/cypress/pull/20722/commits/b06f21635af698d8d7a006e01ac1d04a699a4b42)